src/PIL/ImageStat.py

```diff
@@ -53,17 +53,23 @@ def _getextrema(self):
         """Get min/max values for each band in the image"""
 
         def minmax(histogram):
-            n = 255
-            x = 0
+            res_min, res_max = 255, 0
             for i in range(256):
                 if histogram[i]:
-                    n = min(n, i)
-                    x = max(x, i)
-            return n, x  # returns (255, 0) if there's no data in the histogram
+                    res_min = i
+                    break
+            for i in range(255, -1, -1):
+                if histogram[i]:
+                    res_max = i
+                    break
+            if res_max >= res_min:
+                return res_min, res_max
+            else:
+                return (255, 0)
```

Comment:
Maybe I'm missing something basic, but isn't `res_max >= res_min` only going to be false if every `histogram[i]` is falsy? In which case, `res_min, res_max` will be `(255, 0)`, so you could just
```suggestion
            return res_min, res_max
```